stock_partner_lot/tests/test_stock_partner_lot.py

```diff
@@ -17,8 +17,9 @@ def setUp(self):
         self.product_2 = self.env.ref('product.product_product_6')
 
         res_users_purchase_user = self.env.ref('purchase.group_purchase_user')
-        user = self.env['res.users'].with_context\
-        ({'no_reset_password': True, 'mail_create_nosubscribe': True})
+        user = self.env['res.users'].with_context(
+            {'no_reset_password': True, 'mail_create_nosubscribe': True}
+        )
         self.purchase_user = user.create({
```

Comment:
this code unused i think `self.purchase_user`.

Reply:
If you plan to use this user to create data you should add `...sudo(self.purchase_user).create...` at [L41](https://github.com/agungrachmatulah/stock-logistics-warehouse/blob/01cc9dcae79ff62edf4e7bca57be9a425060ffc1/stock_partner_lot/tests/test_stock_partner_lot.py#L41)